test/decl/ext/extensions.swift

```diff
@@ -81,10 +81,10 @@ protocol P1 {}
 
 protocol P2 {}
 
-extension () {} // expected-error {{non-nominal type '()' cannot be extended}} {{educational-notes=nominal-types}}
+extension () {} // expected-error {{type '()' cannot be extended}} {{educational-notes=nominal-types}}
 
 typealias TupleAlias = (x: Int, y: Int)
-extension TupleAlias {} // expected-error{{non-nominal type 'TupleAlias' (aka '(x: Int, y: Int)') cannot be extended}} {{educational-notes=nominal-types}}
+extension TupleAlias {} // expected-error{{typealias type 'TupleAlias' (aka '(x: Int, y: Int)') cannot be extended}} {{educational-notes=nominal-types}}
```

Comment:
Can this print the kind of the desugared type instead? A typealias can be extended if the underlying type is a class etc.

Reply:
Yeah, it could if we grab the type kind of the canonical type instead. I have done that now.